testOauth2/go.mod

```diff
@@ -0,0 +1,15 @@
+module testOauth2
+
+go 1.16
+
+require (
+	github.com/antigloss/go v0.0.0-20201201072909-f29271b13566
+	github.com/gofiber/fiber/v2 v2.6.0
+	github.com/gofiber/template v1.6.7
+	github.com/joho/godotenv v1.3.0
+	github.com/klauspost/compress v1.11.12 // indirect
+	github.com/segmentio/encoding v0.2.14
+	github.com/valyala/fasthttp v1.22.0 // indirect
```

Comment:
Please run `go mod tidy` to delete `indirect` dependencies.

Reply:
Impossible to get rid of indirect dependencies - they are brought up by fiber, by the json encoder and by fasthttp.
Gladly taking any suggestion ...